packedForest/src/forestTypes/unsupervisedForests/urerf/splitURerF.h

```diff
@@ -0,0 +1,126 @@
+#ifndef splitURerF_h
+#define splitURerF_h
+
+#include "splitURerFInfo.h"
+#include "../../../baseFunctions/pdqsort.h"
+#include <iostream>
+#include <vector>
+#include <cmath>
+#include <algorithm>
+#include <limits>
+#include <assert.h>
+#include <utility>
+namespace fp{
+	template<typename T>
+		class splitURerF{
+			protected:
+				std::vector<T> featureValsVec;
+
+				static T computeSampleVariance(const double mean, const std::vector<T>& v){
+					double accum = 0.0;
+					std::for_each (std::begin(v), std::end(v), [&](const double d) {
+    						accum += (d - mean) * (d - mean);
+					});
+    					return accum;
+				}
+
+				inline void createData(const std::vector<T> featureVals){
+					featureValsVec.clear();
+					auto siz_vec = featureVals.size();
+					for(unsigned int i=0; i<siz_vec; ++i){
+						featureValsVec.push_back(featureVals.at(i));
+					}
+					//std::sort(featureValsVec.begin(), featureValsVec.end());
+				}
+
+			public:
+				inline splitURerFInfo<T> twoMeanSplit(const std::vector<T>& featureVal, const std::vector<int>& featureNums){
```

Comment:
I've got some weird formatting going on here. Fix by using tabs please.
In vim using `:set list` will show you the differences.

Reply:
Fixed in 5f65da0